library/tinkerpop/pom.xml

```diff
@@ -91,8 +91,13 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-
     <build>
+        <testResources>
+            <testResource>
+                <directory>src/test/resources</directory>
+                <filtering>true</filtering>
```

Comment:
What does this do?

Reply:
This change goes with the change to `gafferpop-tinkerpop-modern.properties`. It runs on the test resources and expands any Maven properties. I don't see why this is required.